### DISCUSSION TITLE
为什么基于成功的开源项目的商业产品会失败？ - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#108] [Announcements]
# 为什么基于成功的开源项目的商业产品会失败？ - 「开源之道」

https://opensourceway.community/posts/opensource_culture/why_product_failed_that_based_successful_open_source_project/

**Comment by ZoomQuiet** [comment]
> ... 四处打听

这其实已经给这个项目判了死刑, 不过好消息是, 此时项目成败的职责已经从技术道路上分离了, 完全由老板自己扛了;
当然, 锅肯定是技术最后背的, 这是必须的, 也就是说, 从那一刻开始, 工资就已经转变为 窝囊费 为主了;

开源到底是什么? 有很多系列文章了, 但是, 开源肯定不是一种成功的商业模型, 毕竟这其中并没有商业元素在, 商业收益都是在开源行为之外的周边效应中挖掘/转换/兑换/替换/...出来的;

想拿开源作为一个广告宣传点没问题, 但是, 想拿开源生产过程来作生意就有点儿天真了;

这类故事, 都是因为技术太过叫真引发的, 老板都都没有错, 问题在 技术竟然真的拿开源开发过程来套商业项目了...

这方面 Odoo 社区反而好点儿, 各个 ERP 厂商, 都是咬定了一个固定版本的 Odoo 完成了单向 fork 后,
开始在一个垂直领域发力, 不再反馈上游, 专心经营自己的市场;
嘦可以及时有效的 cover 住领域客户需求,
这就是一个以成功开源产品为起点的商业市场;

其它的在团队生存下来之前, 先嫑考虑了吧.

**Reply by lijiangsheng1** [reply]
这里一定有观念上的差异。
